qemu/tests/tpm_check_buffer_size.py

```diff
@@ -0,0 +1,54 @@
+import os
+from shutil import rmtree
+
+from avocado.utils import git
+from avocado.utils import process
+
+from virttest import error_context
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Run Check size of buffer before accessing it in libtpms
+    upstream test tree
+
+    1. Download and install libtpms-devel rpm
+    2. Clone libtpms test tree
+    3. Comple and execute test cases
+
+    :param test: QEMU test object.
+    :param params: Dictionary with the test parameters.
+    :param env: Dictionary with test environment.
+    """
+
+    s, o = process.getstatusoutput("rpm -q libtpms-devel", shell=True)
```

Comment:
```suggestion
    s = process.system("rpm -q libtpms-devel", shell=True)
```
Or utils_packages.package_install to try to install it.

Reply:
Updated.